sections/quick-order-form.liquid

```diff
@@ -107,10 +107,10 @@
             <tbody>
               {%-
                 render 'quick-order-product-row',
-                item: collections['shop-all'].products.first,
-                image: collections['shop-all']products.first.featured_media,
+                item: collections['shop-all'].products[5],
+                image: collections['shop-all']products[5].featured_media,
               -%}
-              {%- for variant in collections['shop-all'].products.first.variants -%}
+              {%- for variant in collections['shop-all'].products[5].variants -%}
                 {%-
                   render 'quick-order-list-row',
                   item: variant,
```

Comment:
@melissaperreault @eugenekasimov Have we thought about how this looks like with qty rules / vol pricing? 👀 

The hover adds the the background color change, but that also happens when there is a qty rule/vol pricing change

https://screenshot.click/11-49-o57yy-ck12j.mp4

Just wanted to point out to ensure

Reply:
@melissaperreault so, I can remove it for the QOF, is it right?

Reply:
I think only if there are qty rules or vol pricing 👀

Reply:
But that could change per row 🤔 (because it depends on the variant). So maybe if one variant has it, none of them have this behaviour.. Not sure

Reply:
I don't see any reason to keep the previous hover effect for the qty rules/volume-pricing since we add hover effect for the whole row. Am I missing your point Sofia?